packages/example-corvair/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
`generator-react-server`?  `react-server-example`?

I get that [yeoman](http://yeoman.io/) requires a package that starts with `generator-`, but in general isn't the monorepo convention to have packages with a common prefix?